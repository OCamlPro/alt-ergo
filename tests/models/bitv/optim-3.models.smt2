; Test constrained optimization
(set-logic ALL)
(set-option :produce-models true)
(declare-const x (_ BitVec 32))
(maximize x)
(assert (bvslt x ((_ int2bv 32) (- 1))))
(check-sat)
(get-model)
(get-objectives)
