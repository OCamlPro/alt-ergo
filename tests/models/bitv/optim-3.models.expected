
unknown
(
  (define-fun x () (_ BitVec 32) #xfffffffe)
)
(objectives 
  (x #xfffffffe)
)